src/Service.Host/client/src/actions/fetchMenuActions.js

```diff
@@ -4,7 +4,7 @@ import * as actionTypes from './index';
 
 const fetchMenu = () => ({
     [CALL_API]: {
-        endpoint: `${config.appRoot}/intranet/menu`,
+        endpoint: `https://app.linn.co.uk/intranet/menu-no-auth`,
```

Comment:
change back to ${config.appRoot} ?

Reply:
I wasn't sure if there was a point since this route never works locally (or on sys). Might be worth getting it working on sys though,